### PR TITLE
Add contributing guidelines

### PR DESCRIPTION
**Pull request by tohrnii**

### CODE REVIEW

**Review by Overcastan — APPROVED** [review]
Looks good! Thank you!

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ The project is organized into several crates as follows:
 | [IR](ir) | Contains the IR for AirScript, `AirIR`. `AirIR` is initialized with an AirScript AST, which it converts to an internal representation that can be optimized and used to generate code in multiple target languages. |
 | [Winterfell code generator](codegen/winterfell/) | Contains a code generator targeting the [Winterfell prover](https://github.com/novifinancial/winterfell) Rust library. The Winterfell code generator converts a provided AirScript `AirIR` into Rust code that represents the AIR as a new custom struct that implements Winterfell's `Air` trait. |
 
+## Contributing to AirScript
+
+If you are interested in contributing to AirScript, please have a look at our [Contribution guidelines](https://github.com/0xPolygonMiden/air-script/blob/main/CONTRIBUTING.md). If you want to work on a specific issue, please add a comment on the github issue indicating you are interested before submitting a PR. This will help avoid duplicated effort. If you have thoughts on how we can improve AirScript, please open an issue to discuss.
```
Comment (bobbinth) [inline README.md:28]: Looks good! A couple of small suggestions:

I would add one more sentence in the beginning. Something like:

"AirScript is an open project and we welcome everyone to contribute! If you are interested in contributing ... "

Also, I'd change the last sentence to something like:

"If you have thoughts on how to improve AirScript, we'd love to know them. So, please don't hesitate to open issues."

Also, nit: let's spell "github" as "GitHub".

**Review by bobbinth — APPROVED** [review]
All looks good! Thank you!

**Review by Al-Kindi-0 — APPROVED** [review]
Agree with @bobbinth 's suggestions. Otherwise, looks good to me. Thanks!

**Pull request merged**